PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -171,8 +171,38 @@
                 }
             }
 
-            if view is UILabel {
-                if isTextInputSensitive(view) {
+            if let label = view as? UILabel {
+                if isLabelSensitive(label) {
+                    maskableWidgets.append(view.toAbsoluteRect(parent))
+                    return
+                }
+            }
+
+            if let webView = view as? WKWebView {
+                // since we cannot the webview content, if masking texts or images are enabled
+                // we mask the whole webview as well
+                if isAnyInputSensitive(webView) {
+                    maskableWidgets.append(view.toAbsoluteRect(parent))
+                    return
+                }
+            }
+
+            if let button = view as? UIButton {
+                if isButtonSensitive(button) {
+                    maskableWidgets.append(view.toAbsoluteRect(parent))
+                    return
+                }
+            }
+
+            if let theSwitch = view as? UISwitch {
+                if isSwitchSensitive(theSwitch) {
+                    maskableWidgets.append(view.toAbsoluteRect(parent))
+                    return
+                }
+            }
+
+            if let picker = view as? UIPickerView {
```

Comment:
Added a few missing items that are placeholders for wireframes but not for screenshots and can be sensitive